node-server/src/routing/routing.ts

```diff
@@ -0,0 +1,11 @@
+import { Router } from 'express';
+
+import InitialController from '../controllers/initialController';
+import errorHandler400 from '../errorHandlers/error400';
+import ROUTES from '../enums/routingEnums';
+
+const router = Router();
+router.get(`${ROUTES.greetings}/:name`, InitialController.greetings);
```

Comment:
I think that in our case `name` should be interpreted as a query string parameter?
```suggestion
router.get(`${ROUTES.greetings}?name=:name`, InitialController.greetings);
```
And example of the router: `/api/greetings?name=world`

Regarding `InitialController.greetings`, rewrite class to the function without InitialController. Only `greetings` function.